doc/source/whats_new.rst

```diff
@@ -44,8 +44,8 @@ Changelog
    - Add methods ``get_channel_positions``, ``set_channel_positions`` for setting channel positions,
      and ``plot_projs_topomap`` to ``Raw``, ``Epochs`` and ``Evoked`` objects by `Teon Brooks`_
 
-   - Add EEG bad channel interpolation method (based on spherical splines) to ``Raw``, ``Epochs`` and ``Evoked`` objects
-     by `Denis Engemann`_
+   - Add EEG and MEG bad channel interpolation method (based on spherical splines) to ``Raw``, ``Epochs`` and ``Evoked`` objects
+     by `Denis Engemann`_ and `Mainak Jas`_
```

Comment:
So maybe `Add channel interpolation for EEG (spherical splines) and MEG (field interpolation) to ...`